mozilla-release/gfx/thebes/gfxTextRun.cpp

```diff
@@ -3192,7 +3187,7 @@ already_AddRefed<gfxFont>
 gfxFontGroup::WhichSystemFontSupportsChar(uint32_t aCh, uint32_t aNextCh,
                                           Script aRunScript)
 {
-    gfxFontEntry *fe = 
+    gfxFontEntry *fe =
```

Comment:
This is how normal text editor must work (have no idea what guys from FF usually use) :)